test/integration/rhel-8.3/README

```diff
@@ -0,0 +1 @@
+4.18.0-187.2.el8
```

Comment:
This file is created here and deleted in a later commit. I guess it should not have been created at all?

Reply:
I guess this sequence looks a mess at this point if you review that PR commit by commit. The first 3 commits have been done months ago, while 8.3 was in full development. That README as part of the original WIP PR (deleted by the recent update) was meant to remind me on which build I started that port, which was not 240 at that time. The first 3 commits were rebased to current code and the last 2 commits were added just recently. The reason why it's not technically rebased from RHEL-8.2 is because "most" of the work was already done back then and before we created to port for 8.2, but I wanted to refresh that work with changes we've made to the integration tests since then (it ends up being the same in the end I suppose).

I will squash those commits since the history of what was done in this case, especially way back then doesn't add much value I think.

Hope this makes sense.